homeassistant/components/update/__init__.py

```diff
@@ -208,6 +208,20 @@ def entity_category(self) -> EntityCategory | None:
             return EntityCategory.CONFIG
         return EntityCategory.DIAGNOSTIC
 
+    @property
+    def entity_picture(self) -> str | None:
+        """Return the entity picture to use in the frontend.
+
+        Update entities return the brand icon based on the integration
+        domain by default.
+        """
+        if self.platform is None:
+            return None
+
+        return (
+            f"https://brands.home-assistant.io/{self.platform.platform_name}/icon.png"
```

Comment:
Should we use the fallback variant here in case the brands icon does not exist?

Reply:
🤷  It will now render empty instead. This simply doesn't look so great:

<https://brands.home-assistant.io/_/fnsdhjfskfgsdfghsdfgkdlghd/icon.png>

Reply:
Sure, but empty does not look that great as well if all others have something

Reply:
True!

```suggestion
            f"https://brands.home-assistant.io/_/{self.platform.platform_name}/icon.png"
```